### PR TITLE
🌐 Add Ukrainian translation for `docs/uk/docs/fastapi-cli.md`

### PR DESCRIPTION
**Pull request by ykertytsky**
Hey guys! This would be my first (hopefully) successful  contribution to fastapi docs :)

### CODE REVIEW

File: docs/uk/docs/fastapi-cli.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+За замовчуванням, **автоматичне перезавантаження** увімкнене, автоматично перезавантажуючи сервер кожного разу, коли Ви змінюєте Ваш код. Це ресурсо-завтратно, та може бути менш стабільним, ніж коли воно вимкнене. Ви повинні використовувати його тільки під час розробки. Воно також слухає IP-адресу `127.0.0.1`, що є IP Вашого девайсу для самостійної комунікації з самим собою (`localhost`).
+
+## `fastapi run`
+
+Виконання `fastapi run` запустить FastAPI у продакшн-режимі за замовчуванням.
+
+За замовчуванням, **автоматичне перезавантаження** вимкнене. Воно також прослуховує IP-адресу `0.0.0.0`, що означає всі доступні IP адреси, тим самим даючи змогу будь-кому комунікувати з девайсом. Так ви зазвичай будуте запускати його у продакшн, наприклад у контейнері.
+
+В більшості випадків ви можете (і маєте) мати "termination proxy"  який обробляє HTTPS для вас, це залежить від способу розгортання вашого додатку, ваш провайдер може зробити це для вас, або вам потрібно налаштувати його самостійно.
```
Comment (rostik1410) [inline docs/uk/docs/fastapi-cli.md:77]:
```suggestion
В більшості випадків Ви можете (і маєте) мати "termination proxy", який обробляє HTTPS для вас, це залежить від способу розгортання вашого додатку, Ваш провайдер може зробити це для вас, або вам потрібно налаштувати його самостійно.
```

**Review by rostik1410 — CHANGES_REQUESTED** [review]

File: docs/uk/docs/fastapi-cli.md (hunk 1 above)
Comment (rostik1410) [reply docs/uk/docs/fastapi-cli.md:77]: @ykertytsky Great job. There are just several typos.

File: docs/uk/docs/fastapi-cli.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+/// tip
+
+Ви можете дізнатись більше про це у [документації для розгортування](deployment/index.md){.internal-link target=_blank}.
```
Comment (mezgoodle) [inline docs/uk/docs/fastapi-cli.md:81]:
```suggestion
Ви можете дізнатись більше про це у [документації про розгортування](deployment/index.md){.internal-link target=_blank}.
```

**Review by mezgoodle — COMMENTED** [review]
@ykertytsky , great job! Agree with @rostik1410 , apply his suggestions please.

**Comment by ykertytsky** [comment]
> @ykertytsky , great job! Agree with @rostik1410 , apply his suggestions please.

Thank you! Critique is taken to account, will fix mentioned issues shortly.

**Comment by ykertytsky** [comment]
Okay, @rostik1410 @mezgoodle all mentioned typos and mistakes corrected succesfully🫡

**Review by mezgoodle — APPROVED** [review]

**Review by rostik1410 — APPROVED** [review]

**Comment by ykertytsky** [comment]
@alejsdev Hey! I am pretty sure we are done here :)

**Comment by rostik1410** [comment]
> @alejsdev Hey! I am pretty sure we are done here :)

sometimes a merge takes a lot of time, don't worry about it)

**Comment by alejsdev** [comment]
Thanks for your contribution! @ykertytsky :nerd_face: 
And thanks for the help @mezgoodle @rostik1410 :100:

**Pull request merged**